test/test.js

```diff
@@ -21071,6 +21071,95 @@
   }());
 
   /*--------------------------------------------------------------------------*/
+  QUnit.module('lodash.floor');
```

Comment:
There are already tests for these methods: https://github.com/tejokumar/lodash/blob/bug/1478/test/test.js#L15784

I don't think a new block is necessary, just a new assertion template.